static/lang/en.json

```diff
@@ -93,7 +93,8 @@
     "header": "Fissures",
     "voidstorm": "Void Storm",
     "choice": "Choose which fissures to show",
-    "both": "Both"
+    "both": "All",
```

Comment:
nit picky, but perhaps change the key as well?

Reply:
sure
```suggestion
    "all": "All",
```